packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2206,6 +2206,14 @@ export default class Meeting extends StatelessWebexPlugin {
 
       LoggerProxy.logger.log('Meeting:index#setLocalAudioTrack --> Audio settings.', JSON.stringify(this.mediaProperties.mediaSettings.audio));
       this.mediaProperties.setLocalAudioTrack(audioTrack);
+
+      this.mediaProperties.audioTrack.onmute = (e) => {
+        console.warn('muted', e);
```

Comment:
Is this meant to be here? or are they supposed to be logs?

Reply:
cleaned up